packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -0,0 +1,192 @@
+import { MoreVertical } from "@comet/admin-icons";
+import {
+    Button,
+    Chip,
+    ChipProps,
+    Divider,
+    DividerProps,
+    ListItemIcon,
+    ListItemText,
+    Menu,
+    MenuItem,
+    MenuList,
+    MenuListProps,
+    MenuProps,
+    Typography,
+    TypographyProps,
+} from "@mui/material";
+import { Maybe } from "graphql/jsutils/Maybe";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+function CrudMoreActionsDivider(props: DividerProps) {
+    return <Divider sx={{ margin: "8px 10px", borderColor: (theme) => theme.palette.grey[50] }} {...props} />;
+}
+
+interface CrudMoreActionsGroupProps {
+    groupTitle: React.ReactNode;
+    menuListProps?: MenuListProps;
+    typographyProps?: React.ComponentProps<typeof Typography>;
+}
+
+function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
+    return (
+        <>
+            <Typography variant="subtitle2" color={(theme) => theme.palette.grey[500]} fontWeight="bold" pt="20px" px="15px" {...typographyProps}>
+                {groupTitle}
+            </Typography>
+            <MenuList {...menuListProps}>{children}</MenuList>
+        </>
+    );
+}
+
+export interface ActionItem extends React.ComponentProps<typeof MenuItem> {
+    type: "action";
+    label: React.ReactNode;
+    startAdornment?: React.ReactNode;
+}
+
+export interface DividerItem extends React.ComponentProps<typeof Divider> {
+    type: "divider";
+}
+
+type CrudMoreActionsItem = ActionItem | DividerItem;
+
+export interface CrudMoreActionsMenuProps {
+    selectionSize?: number;
+    buttonProps?: React.ComponentProps<typeof Button>;
+    menuProps?: Partial<MenuProps>;
+    chipProps?: Partial<ChipProps>;
+    groupTypographyProps?: Partial<TypographyProps>;
+    overallItems?: Maybe<CrudMoreActionsItem>[];
+    selectiveItems?: Maybe<CrudMoreActionsItem>[];
```

Comment:
Because there now were divider and an action items. If I implement the divider like you said, I will rename it to overallActions and selectiveActions 👍🏻